src/GraphQL/ResolverRegistryInterface.php

```diff
@@ -6,6 +6,7 @@
 use Drupal\graphql\GraphQL\Execution\ResolveContext;
 use Drupal\graphql\GraphQL\Resolver\ResolverInterface;
 use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
```

Comment:
this should be removed now.

Reply:
That's what I get for multi-tasking. Thanks for doing the difficult parts of programming for me 🙇‍♂️ 😅